charts/nr-k8s-otel-collector/README.md

```diff
@@ -1,75 +1,107 @@
-# Installation: 
+[![Community Plus header](https://github.com/newrelic/opensource-website/raw/master/src/images/categories/Community_Plus.png)](https://opensource.newrelic.com/oss-category/#community-plus)
 
-### 1. Download source code 
-```
-git clone https://github.com/newrelic/helm-charts.git
-```
+# nr-k8s-otel-collector
+
+A Helm chart to monitor a Kubernetes Cluster using an OpenTelemetry Collector.
+
+# Helm installation
 
-### 2. Update config [here](https://github.com/newrelic/helm-charts/tree/master/charts/nr-k8s-otel-collector/values.yaml#L20-L24) to add a cluster name, and New Relic Ingest - License key
-Example: 
+Download and Update config [here](https://github.com/newrelic/helm-charts/tree/master/charts/nr-k8s-otel-collector/values.yaml#L20-L24) to add a cluster name, and New Relic Ingest - License key
+
+Example:
 ```
 licenseKey: "EXAMPLEINGESTLICENSEKEY345878592NRALL"
-cluster: "SampleApp" 
+cluster: "SampleApp"
 ```
 
-### 3. From the root directory of this chart, run:
-```
-helm install nr-k8s-otel-collector nr-k8s-otel-collector -n newrelic --create-namespace
+You can install this chart using directly this Helm repository:
+
+```shell
+helm repo add newrelic https://helm-charts.newrelic.com
+helm upgrade nr-k8s-otel-collector newrelic/nr-k8s-otel-collector -f your-custom-values.yaml  -n newrelic --create-namespace  --install
 ```
 
 ## Confirm installation
-### Watch pods spin up: 
+### Watch pods spin up:
+
 ```
-kubectl get pods -A --watch 
+kubectl get pods -n newrelic --watch
 ```
 
-### Check logs of opentelemetry pod that spins up: 
+### Check logs of opentelemetry pod that spins up:
 ```
 kubectl logs <otel-pod-name> -n newrelic
 ```
 
-### Confirm data coming through in New Relic 
+### Confirm data coming through in New Relic
 You should see data reporting into New Relic within a couple of seconds to the `InfrastructureEvent` table, `Metric` table, and `Log` tables.
 ```
-FROM Metric SELECT * 
+FROM Metric SELECT * WHERE k8s.cluster.name='<CLUSTER_NAME>'
 ```
 ```
-FROM InfrastructureEvent SELECT * 
+FROM InfrastructureEvent SELECT * WHERE k8s.cluster.name='<CLUSTER_NAME>'
 ```
 ```
-FROM Log SELECT * 
+FROM Log SELECT * WHERE k8s.cluster.name='<CLUSTER_NAME>'
 ```
```

Comment:
maybe we want to add a section on how to `uninstall` the collector? i know thats not included in the other agents, but just a suggestion, i'm happy w/ it either way. 

`helm uninstall nr-k8s-otel-collector -n newrelic` if the earlier command was followed